<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBEpdEditWidget</class>
 <widget class="QWidget" name="SVDBEpdEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>846</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Construction editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelUUID">
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditUUID"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="labelCategory_2">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCategory">
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditCategory"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelManufacturer">
     <property name="text">
      <string>Manufacturer:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditManufacturer"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelExpireYear">
     <property name="text">
      <string>Expire year:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditExpireYear"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data source:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelRefUnit">
     <property name="text">
      <string>Reference Unit:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditRefUnit"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelRefQuantity">
     <property name="text">
      <string>Reference Quantity:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditRefQuantity"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidgetEpdData"/>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButtonAddEpd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Adds Epd Module</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SIM-VICUS.qrc">
         <normaloff>:/gfx/actions/24x24/plus.png</normaloff>:/gfx/actions/24x24/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRemoveEpd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Removes currently selected Module from EPD</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/SIM-VICUS.qrc">
         <normaloff>:/gfx/actions/24x24/minus.png</normaloff>:/gfx/actions/24x24/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/SIM-VICUS.qrc"/>
 </resources>
 <connections/>
</ui>
